examples/saul/Makefile

```diff
@@ -0,0 +1,28 @@
+# name of your application
+APPLICATION = default
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# we want to use SAUL:
+USEMODULE += saul_reg
+USEMODULE += saul_default
```

Comment:
its not. But I think it does make sense to break everything down to simply including `saul_default`. If we ever find a use-case, where this doesn't work, I'd say we can always revert this decision... So I will push that change.